about.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About us</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+    <script
+      src="https://kit.fontawesome.com/7e86c8c237.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <nav id="toolbar">
+        <div class="mob-nav-container">
+          <a class="nav-bars">
+            <i class="fa-sharp fa-solid fa-bars fs-1 fw-bold"></i>
+          </a>
+        </div>
+        <ul id="toolbar-items" class=" d-none list-unstyled justify-content-end">
+          <i class="fa-brands fa-facebook-f p-2"></i>
+          <i class="fa-brands fa-twitter p-2"></i>
+          <li class="toolbar-item p-2">English</li>
+          <li class="toolbar-item p-2">My Page</li>
+          <li class="toolbar-item p-2">Logout</li>
+        </ul>
+        <div class="nav-items d-none list-unstyled justify-content-between px-4">
+          <div class="logo d-flex flex-row align-content-center justify-content-center align-items-center">
+            <img class="logo-img" src="./assets/icons/logo.png" alt="logo">
+            <div class="vertical-bord"></div>
+            <a href="./" class="carento text-decoration-none fs-2 fw-bold">Carento.</a>
+          </div>
+          <ul class="nav-btns d-flex list-unstyled justify-content-end align-content-center align-items-center m-0">
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="./">Home</a></li>
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="./about.html">About</a></li>
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="">Join</a></li>
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="">Sponsor</a></li>
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="">News</a></li>
+            <li class="nav-item special p-2 px-4"><a class="text-decoration-none text-dark fw-normal" href="">Our Offers</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main class="about-us bg-light">
+        <section class="hero d-flex flex-column align-content-center align-items-center ">
+            <h1 class="title-about text-center ">Creative Cars Renting Website 2022</h1>
```

Comment:
- [ ] A whole section is missing here.
### Your Page
![image](https://user-images.githubusercontent.com/45460192/190657434-2eea6c2b-0d40-4fba-b25e-5657866e24b7.png)

### Required
![image](https://user-images.githubusercontent.com/45460192/190657588-7259bc18-44c5-4e8a-a99d-fc1fa82e9dfe.png)